roles/analysis/tasks/main.yml

```diff
@@ -1,5 +1,11 @@
 ---
 # tasks file for analysis
+- name: Check distribution and install convert2rhel
+  ansible.builtin.import_role:
+    name: infra.convert2rhel.common
+  vars:
+    job_name: 'convert2rhel analysis'
```

Comment:
Not sure if this is needed - similar variable is in leapp collection when common role is included https://github.com/redhat-cop/infra.leapp/blob/main/roles/analysis/tasks/main.yml#L7

Reply:
Removed it